M/Makemsix/build_tarballs.jl

```diff
@@ -0,0 +1,159 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Makemsix"
+version = v"1.7.241"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/microsoft/msix-packaging.git",
+              "efeb9dad695a200c2beaddcba54a52c8320bd135"),
+    DirectorySource(joinpath(@__DIR__, "patches"))
+]
+
+
+# Script that will adapt to each platform
+script = raw"""
+    cd $WORKSPACE/srcdir/msix-packaging
+
+    # Update C++ standard to 17
+    sed -i 's/set(CMAKE_CXX_STANDARD 14)/set(CMAKE_CXX_STANDARD 17)/' CMakeLists.txt
+    find . -name "CMakeLists.txt" -type f -exec sed -i 's/set(CMAKE_CXX_STANDARD 14)/set(CMAKE_CXX_STANDARD 17)/' {} \;
+    find . -name "CMakeLists.txt" -type f -exec sed -i 's/cmake_minimum_required(VERSION 3.29.0 FATAL_ERROR)/cmake_minimum_required(VERSION 3.21.7)/' {} \;
+
+    cp ../lib/CMakeLists.txt lib/CMakeLists.txt
+    cp ../src/msix/PAL/Signature/OpenSSL/SignatureValidator.cpp src/msix/PAL/Signature/OpenSSL/SignatureValidator.cpp 
```

Comment:
Any chance for this code not to live here?

Reply:
Eventually, I will condense it into a patch. The changes are necessary to eliminate the use of OpenSSL internal API functions. 

The original `lib/CMakeLists.txt` was written with vendored dependencies in mind and required significant work to compile with external dependencies.

I am currently trying to port from MSVC to the Mingw compiler for Windows compilation. If successful, this will also involve significant patching.